### DISCUSSION TITLE
Ready to release SpamBlocker with PhoneBlock(production API)

### DISCUSSION THREAD

**Discussion by aj3423** [haumacher/phoneblock#121] [General]
Hi.
I'm ready to publish the new SpamBlocker, it's been tested by a few people, 10 downloads per day. People usually don't download from the latest tag, but for the new release, there will be much more downloads. 

If you are ready, I'll release it tomorrow with the production API.

**Comment by haumacher** [comment]
Latest release I installed (3.5) has still the pre-configured test API included. Are there still problems?

**Reply by aj3423** [reply]
So far, there is no problem. But only 50+ people have installed that pre-release. 
I'll release the v4.0 after a couple of hours, with the production API.

**Reply by aj3423** [reply]
The v4.0 is live :)
